src/components/profile/header/header.profile.jsx

```diff
@@ -0,0 +1,44 @@
+import React, { useState } from "react";
+import "./header.profile.scss";
+import mainProfilePicture from "./profile1.jpeg";
+import searchIcon from "./mag.png";
+import addFriendIcon from "./add.png";
+
+function Header({ onSearchChange }) {
+  let [searchToggle, setSearchToggle] = useState(false);
+  return (
+    <>
+      <div className="Header">
+        <h1>친구</h1>
+        {searchToggle === true ? (
+          <input placeholder="검색" onChange={onSearchChange} />
+        ) : null}
```

Comment:
```suggestion
        {searchToggle && <input placeholder="검색" onChange={onSearchChange} />}
```
[단축 평가 논리 계산법](https://velog.io/@nudge411/%EB%8B%A8%EC%B6%95-%ED%8F%89%EA%B0%80-short-circuit-evaluation-%EB%85%BC%EB%A6%AC-%EA%B3%84%EC%82%B0%EB%B2%95)
이런 식으로 더 간단하게 써 볼 수 있습니당